compiler/scripts/settings.js

```diff
@@ -19,12 +19,11 @@ async function getSourceSettings(writeKey) {
     request.get({ url: url, gzip: true }, (err, _, body) => {
       if (err) {
         reject(err);
+      } else if (body && body.includes('Invalid path or write key provided.')) {
+        console.error('Please make sure your Segment writeKey was entered correctly.')
```

Comment:
"More info: https://segment.com/docs/connections/find-writekey"

Reply:
Fancy. I like it.